frontend/templates/base.html

```diff
@@ -1,22 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="{{ url_for('static', filename='general.css') }}">
-    <title>Mastermind - {% block title %}{% endblock title %}</title>
-    <body>
 
-      <h1>{% block header1 %}{% endblock header1 %}</h1>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="{{ url_for('static', filename='general.css') }}">
+  <title>Mastermind - {% block title %}{% endblock title %}</title>
+  {% block stylesheets %}{% endblock stylesheets %}
+</head>
+<body>
```

Comment:
I just separated the `<head>` and `<body>` here because previously the `body` was inside the `head`, which is invalid HTML